sky/serve/core.py

```diff
@@ -94,14 +94,17 @@ def _validate_service_task(task: 'sky.Task') -> None:
 def up(
     task: 'sky.Task',
     service_name: Optional[str] = None,
-) -> None:
+) -> Tuple[str, str]:
     """Spin up a service.
 
     Please refer to the sky.cli.serve_up for the document.
 
     Args:
         task: sky.Task to serve up.
         service_name: Name of the service.
+
+    Returns:
+        A tuple with two values: the service name and the endpoint.
```

Comment:
nit: is it possible to use a similar format with https://github.com/skypilot-org/skypilot/blob/5a2f1b855e0a8b0a099390882209914f34ea8b98/sky/execution.py#L444-L449

Also cc @Michaelvll for discussion: do you think it would be helpful to return the controller handle, like `sky.launch`?

Reply:
It feels fine to me to only return the service name and the endpoint for now, as the controller handle can always be retrieved with `sky.status`.